lib/browser/init.js

```diff
@@ -162,8 +162,8 @@ require('./api/protocol')
 // Set main startup script of the app.
 const mainStartupScript = packageJson.main || 'index.js'
 
-// Workaround for electron/electron#5050
-if (process.platform === 'linux' && process.env.XDG_CURRENT_DESKTOP === 'Pantheon') {
+// Workaround for electron/electron#5050 and electron/electron#9046
+if (process.platform === 'linux' && ['Pantheon', 'Unity:Unity7'].indexOf(process.env.XDG_CURRENT_DESKTOP) !== -1) {
```

Comment:
Very minor but this could be:

```js
['Pantheon', 'Unity:Unity7'].includes(process.env.XDG_CURRENT_DESKTOP)
```

Which is a little more concise.